### ISSUE TITLE
Using Loading Plugin with QIcon fails

### ISSUE THREAD

**Issue by nicholaszuccarelli** [quasarframework/quasar#15245]
### What happened?

I am attempting to use a custom icon to replace the default QSpinner for the Loading Plugin.

Attempting to do so causes the following error:
**Uncaught TypeError: Cannot read properties of undefined (reading 'iconMapFn')**
I am assuming this happens because it is trying to reference a Quasar instance which is outside of the default createApp() (since Loading has been called outside of the component)

![image](https://user-images.githubusercontent.com/13761766/211442639-12566d03-0fd4-4542-a037-fe924e8ce325.png)

### What did you expect to happen?

(This worked in Quasar 1)
The default spinner should be replaced with the specified QIcon

### Reproduction URL

https://github.com/nicholaszuccarelli/quasar-broken-loading-plugin

### How to reproduce?

-Clone git project
-npm install
-quasar dev
-Click either buttons on the main view and check error in console
--Button 1 creates a separate Loading plugin instance outside of the component
--Button 2 uses the (quasar.conf) "Loading" plugin injected into the instance
Fails on both

### Flavour

Quasar CLI with Webpack (@quasar/cli | @quasar/app-webpack)

### Areas

Plugins (quasar)

### Platforms/Browsers

Chrome

### Quasar info output

```Shell
Operating System - Windows_NT(10.0.22621) - win32/x64
NodeJs - 16.18.1

Global packages
  NPM - 8.19.2
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*

Networking
  Host - NICHOLAS-PCi9
  vEthernet (WSL) - 172.17.32.1
  Ethernet - 192.168.1.50
```

### Relevant log output

```Shell
runtime-core.esm-bundler.js?f781:220 Uncaught TypeError: Cannot read properties of undefined (reading 'iconMapFn')
    at ReactiveEffect.eval [as fn] (QIcon.js?0016:83:1)
    at ReactiveEffect.run (reactivity.esm-bundler.js?a27c:187:1)
    at get value [as value] (reactivity.esm-bundler.js?a27c:1150:1)
    at Proxy.eval (QIcon.js?0016:192:1)
    at renderComponentRoot (runtime-core.esm-bundler.js?f781:898:1)
    at ReactiveEffect.componentUpdateFn [as fn] (runtime-core.esm-bundler.js?f781:5631:1)
    at ReactiveEffect.run (reactivity.esm-bundler.js?a27c:187:1)
    at instance.update (runtime-core.esm-bundler.js?f781:5745:1)
    at setupRenderEffect (runtime-core.esm-bundler.js?f781:5759:1)
    at mountComponent (runtime-core.esm-bundler.js?f781:5541:1)
```

### Additional context

From QIcon.js
![image](https://user-images.githubusercontent.com/13761766/211443089-60e42e5f-f4a5-4d39-a13f-9d9f45bdd491.png)

**Comment by rstoenescu** [comment]
Thank you for reporting!
Fix will be available in Quasar v2.11.5

**Comment by nicholaszuccarelli** [comment]
Great stuff as always! ☺

**Comment by ivanpepelko** [comment]
Something went wrong with the fix commit. When trying to update to via dependabot to 2.11.5, my tests fail with exception:
![image](https://user-images.githubusercontent.com/7328461/212661764-d334611f-44cb-4967-a42e-2cf12e996c5e.png)
![image](https://user-images.githubusercontent.com/7328461/212661931-c444b22a-6f99-414f-bf24-4146644c7d5c.png)

`Loading.show` is called in the `Router.beforeEach` hook. 
I will provide more info upon request.

**Comment by IlCallo** [comment]
@ivanpepelko it appears `Loading` plugin used to work even without registering it into `quasar.config.js > framework > plugins` dedicated option
Just got bit from this too, I'm actually surprised it worked up until now

**Comment by ivanpepelko** [comment]
Can confirm it worked without activating the plugin. However, it stopped working after the fix for current issue was merged (I opened #15287, which was closed promptly due to missing config).
